addons/vagrant/wireless/Vagrantfile

```diff
@@ -30,8 +30,29 @@ Vagrant.configure("2") do |config|
                      :ip => details['mgmt_ip'],
                      :netmask => details['mgmt_netmask'],
                      :libvirt__dhcp_enabled => false,
-                     :libvirt__forward_mode => networks[0]['forward_mode'],
-                     auto_config: true
+                     :libvirt__forward_mode => networks[0]['forward_mode']
+      # link reg_network (vlan 2)
+      srv.vm.network "private_network",
+                     :mac => "44:34:22:11:00:02",
+                     :libvirt__network_name => networks[1]['name'],
+                     :ip => "172.17.2.100",
+                     :netmask => networks[1]['netmask'],
+                     :libvirt__dhcp_enabled => false,
+                     :libvirt__forward_mode => networks[1]['forward_mode']
+      # link iso_network (vlan 3)
+      srv.vm.network "private_network",
+                     :mac => "44:34:22:11:00:03",
+                     :libvirt__network_name => networks[2]['name'],
+                     :ip => "172.17.3.100",
+                     :netmask => networks[2]['netmask'],
+                     :libvirt__dhcp_enabled => false,
+                     :libvirt__forward_mode => networks[2]['forward_mode']
```

Comment:
Is it still necessary ?

Reply:
Since we are not able to map the hostapd dynamic vlan on register vlan, no it is not necessary.
I will remove it.
Thanks